source/agora/common/Config.d

```diff
@@ -538,6 +556,11 @@ private NodeConfig parseNodeConfig (Node* node, in CommandLine cmdln)
     Duration relay_tx_interval = opt!(ulong, "node", "relay_tx_interval_secs")(cmdln, node, 15).seconds;
     const relay_tx_min_fee = Amount(opt!(ulong, "node", "relay_tx_min_fee")(cmdln, node, 0));
     Duration relay_tx_cache_exp = opt!(ulong, "node", "relay_tx_cache_exp_secs")(cmdln, node, 1200).seconds;
+    const collect_network_statistics = opt!(bool, "node", "collect_network_statistics")(cmdln, node, false);
+    const include_in_network_statistics = opt!(bool, "node", "include_in_network_statistics")(cmdln, node, true);
+    const num_of_crawlers = opt!(ubyte, "node", "num_of_crawlers")(cmdln, node, 1);
+    Duration crawling_interval = opt!(ulong, "node", "crawling_interval_secs")(cmdln, node, 3).seconds;
+    const ipdb_path = opt!(string, "node", "ipdb_path")(cmdln, node, "infrastructure/bigfiles/ipdb.mmdb");
```

Comment:
In general, one should use default values on the struct instead of default values here.